Makefile

```diff
@@ -223,6 +223,11 @@ clean: | clean-common
 ifneq ($(USE_LIBBACKTRACE), 0)
 	+ $(MAKE) -C vendor/nim-libbacktrace clean $(HANDLE_OUTPUT)
 endif
+	cargo clean --manifest-path vendor/rln/Cargo.toml
+
+# clean the rln build (forces recompile of old crates on next build)
+cleanrln:
```

Comment:
Thanks for running the test and sharing the issue! 
There is a similar target `clean` which is supposed to clean the old artifacts of the rln compilation, have you tried it? let me know whether that works or not